README.md

```diff
@@ -107,3 +109,7 @@ List of tools curated by Protofire team to better apply best practices to the Pr
 - [Reddit: /r/ethereum/](https://www.reddit.com/r/ethereum/) - Reddit about next-generation platform for decentralised applications.
 - [Reddit: /r/ethdev](https://www.reddit.com/r/ethdev) - Reddit about ethereum-related dev talk: Contracts, DApps, Wallets, Clients, Infrastructure, Tooling, UIs, Patterns, and others.
 - [Ethereum Research](http://ethresear.ch/) - Civilized discussion furthering Ethereum research
+
+#### Talks
```

Comment:
A few items here:
1. I think this doesn't belong to 'Toolbox'.
1. I tend to believe that if we keep adding categories, we'll end up having a ton of categories with just a few items in each of them. Why don't we add a 'Miscellaneous' category and that's it?

@fvictorio @mariano-aguero thoughts?

Reply:
Mmm I agree to go ahead with  `Miscellaneous` category, maybe in another PR we can improve the readme an add other items like `Ethereum vs. Bitcoin` or `Under the hood` inside `Miscellaneous` (thinking out loud)

Reply:
@pablofullana  @mariano-aguero 
It sounds good to me. So, should i change 'Toolbox' category by 'Miscellaneous' or just create a miscellaneous category with the 'talks' section? 

Thank you so much for the feedback!